test/libsolidity/SolidityNameAndTypeResolution.cpp

```diff
@@ -5611,7 +5621,7 @@ BOOST_AUTO_TEST_CASE(using_interface_complex)
 	char const* text = R"(
 		interface I {
 			event A();
-			function f();
+			function f() returns(bool);
```

Comment:
Can't comment on the right spot, but this test should fail. The interace defines a return parameter, while the implementation is omitting it.